### ISSUE TITLE
Difficulty installing on CentOS (Amazon AMI)

### ISSUE THREAD

**Issue by mikepugh** [berkshelf/berkshelf-api#186]
I'm spinning up an Amazon Linux AMI in AWS (CentOS), and I'm having it run the Chef Client to install the Berkshelf API Server.

The runlist is simply "berkshelf-api-server".

Chef reports a successful run, and I see that nginx is running. However when I try to access the endpoint, nginx reports a Bad Gateway (502) error.

I stopped the berks-api service and attempted to run it manually with the debug flag but it appears to fail on a bunch of dependencies.

At first it was telling me:

````bash
# ruby berks-api -d
/usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- buff/extensions (LoadError)
....
````

I tried gem install buff-extensions, and then tried the berks-api again and got

````bash
# ruby berks-api -d
/usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file --celluloid (LoadError)
....
````

I was not successful in installing celluloid via the gem installer.

Any ideas on how to resolve? Thanks

**Comment by mikepugh** [comment]
I got this working with some manual intervention:

````bash
yum -y install gcc ruby-devel rubygems
````
Then I was able to ````gem install berkshelf-api```` and after restarting the service, it started to work.

I did try using this cookbook locally via Vagrant and an Ubuntu 14 box - but ran into the same problem. Same story with a Debian box on Vagrant.

Should the cookbook be modified to include those extra packages?

**Comment by mrmarbury** [comment]
You shouldn't need an extra Ruby. The Ruby and its embedded stuff that comes with chef-client should be enough. Shouldn't it?

**Comment by lamont-granquist** [comment]
probably going to be addressed by #218
